config/vault.go

```diff
@@ -34,6 +34,14 @@ const (
 	// DefaultLeaseRenewalThreshold is the default fraction of a non-renewable
 	// lease to wait for before refreshing
 	DefaultLeaseRenewalThreshold = .90
+
+	// DefaultK8SServiceAccountTokenPath is a default path to a file
+	// with service token for the k8s auth method.
+	DefaultK8SServiceAccountTokenPath = "/var/run/secrets/kubernetes.io/serviceaccount/token"
```

Comment:
Should be /run/secrets/kubernetes.io/serviceaccount/token

Reply:
Thanks for the review @yellowmegaman!

You say LGTM *and* suggest a change... does it need that change first, and then can be merged or is it good without that change and can be merged as is? Thanks.

Reply:
@eikenb @yellowmegaman changed

Reply:
I updated it here and in [envconsul]( https://github.com/hashicorp/envconsul/pull/281/commits/67ee7c33f1ce78b7d34f4335fa001a2ed832ce69)

Reply:
@eikenb sorry for confusion, it was "change needed, but otherwise LGTM"